lzero/worker/muzero_collector.py

```diff
@@ -204,13 +204,13 @@ def _compute_priorities(self, i, pred_values_lst, search_values_lst):
             - pred_values_lst: The list of value being predicted.
             - search_values_lst: The list of value obtained through search.
         """
-        if self.policy_config.use_priority and not self.policy_config.use_max_priority_for_new_data:
+        if self.policy_config.use_priority:
             pred_values = torch.from_numpy(np.array(pred_values_lst[i])).to(self.policy_config.device).float().view(-1)
             search_values = torch.from_numpy(np.array(search_values_lst[i])).to(self.policy_config.device
                                                                                 ).float().view(-1)
             priorities = L1Loss(reduction='none'
                                 )(pred_values,
-                                  search_values).detach().cpu().numpy() + self.policy_config.prioritized_replay_eps
+                                  search_values).detach().cpu().numpy() + 1e-6  # avoid 0 priority
```

Comment:
补充注释：```
# Calculate priorities. The priorities are the L1 losses between the predicted
# values and the search values. We use 'none' as the reduction parameter, which
# means the loss is calculated for each element individually, instead of being summed or averaged. 
# A small constant (1e-6) is added to the results to avoid zero priorities. This
# is done because zero priorities could potentially cause issues in some scenarios.
```